sphinx/source/_static/custom.css

```diff
@@ -1,6 +1,6 @@
 .sphinxsidebarwrapper {
     position: fixed;
     width: 16em;
-    height: 100%;
+    height: 90%;
```

Comment:
This helps avoid overflow (it was impossible to view the entries at the bottom; this ensures that the user can scroll to and view the end.)